lib/base-compiler.ts

```diff
@@ -74,6 +74,7 @@ import {languages} from './languages.js';
 import {LlvmAstParser} from './llvm-ast.js';
 import {LlvmIrParser} from './llvm-ir.js';
 import * as compilerOptInfo from './llvm-opt-transformer.js';
+import {parse} from './stack-usage-transformer.js';
```

Comment:
Just since there's a lot of parsing going on in this file for just about everything, can this be done as `import * as StackUsageTransformer from './stack-usage-transformer.js';` and then `StackUsageTransformer.parse` later